package.json

```diff
@@ -49,7 +49,7 @@
     "eslint-plugin-import": "^1.13.0",
     "eslint-plugin-jsx-a11y": "^2.1.0",
     "eslint-plugin-react": "^6.1.1",
-    "extract-text-webpack-plugin": "3.0.2",
+    "extract-text-webpack-plugin": "^4.0.0-beta.0",
```

Comment:
Consider using `mini-css-extract-plugin` instead of `extract-text-webpack-plugin`. The latter has issues with entries reordering and now is in maintenance mode, see [issue](https://github.com/webpack-contrib/extract-text-webpack-plugin/issues/737#issuecomment-374548804).